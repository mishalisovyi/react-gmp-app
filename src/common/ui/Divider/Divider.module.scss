@use "../../../colors";

.Divider {
  height: 4px;
  position: relative;
  top: -4px;
  background-color: colors.$black-panther;
}
