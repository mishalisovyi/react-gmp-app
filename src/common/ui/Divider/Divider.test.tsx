import React from 'react';
import renderer from 'react-test-renderer';

import { Divider } from './Divider';

it('should render divider', () => {
  const tree = renderer.create(<Divider />);

  expect(tree.toJSON()).toMatchSnapshot();
});
