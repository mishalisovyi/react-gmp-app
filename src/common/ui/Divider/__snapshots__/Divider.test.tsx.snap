// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render divider 1`] = `<div />`;
